.github/workflows/build.yml

```diff
@@ -220,6 +220,7 @@ jobs:
         if [ "$PGROLL_SCHEMA" != "public" ]; then
           psql -h $POSTGRES_HOST -p $POSTGRES_PORT -U postgres -c "CREATE SCHEMA $PGROLL_SCHEMA;"
         fi
+        psql -h $POSTGRES_HOST -p $POSTGRES_PORT -U postgres -c "CREATE EXTENSION IF NOT EXISTS btree_gist SCHEMA $PGROLL_SCHEMA;"
```

Comment:
We need to make a similar change to the local `docker compose` setup too:

```
docker compose up
make examples
```

now fails because the `btree_gist` extension isn't installed.